### ISSUE TITLE
コントラスト比がWCAGの達成基準を超えても常にFailになってしまう

### ISSUE THREAD

**Issue by oti** [oti/hex-crc#22]
#21 のエンバグです。

判定に必要な値を emit していましたがこれをやめたのがミスだったと思われます。
